src/main/java/spireMapOverhaul/zones/manasurge/ManaSurgeZone.java

```diff
@@ -91,206 +92,106 @@ private boolean isPositiveModifier(AbstractCard card) {
+            commonModifierList.addAll(getPositiveCommonModifierList(true));
+            commonModifierList.addAll(getNegativeCommonModifierList(true));
+            AbstractManaSurgeModifier selectedModifier = commonModifierList.get(AbstractDungeon.cardRng.random(commonModifierList.size()));
+            if (selectedModifier != null) {
+                CardModifierManager.addModifier(card, selectedModifier);
             }
         } else {
-            AbstractCardModifier modifier;
-            switch (AbstractDungeon.cardRng.random(4)) {
-                case 0:
-                    modifier = new PowerfulModifier(true);
-                    break;
-                case 1:
-                    modifier = new ProtectiveModifier(true);
-                    break;
-                case 2:
-                    modifier = new BrittleModifier(true);
-                    break;
-                case 3:
-                    modifier = new PowerlessModifier(true);
-                    break;
-                default:
-                    modifier = null;
-                    break;
-            }
-            if (modifier != null) {
-                CardModifierManager.addModifier(card, modifier);
+            List<AbstractManaSurgeModifier> uncommonModifierList = new ArrayList<>();
+            uncommonModifierList.addAll(getPositiveUncommonModifierList(true));
+            uncommonModifierList.addAll(getNegativeUncommonModifierList(true));
+            AbstractManaSurgeModifier selectedModifier = uncommonModifierList.get(AbstractDungeon.cardRng.random(uncommonModifierList.size()));
+            if (selectedModifier != null) {
+                CardModifierManager.addModifier(card, selectedModifier);
             }
         }
     }
 
     public static void applyPermanentPositiveModifier(AbstractCard card) {
         if (AbstractDungeon.cardRng.randomBoolean(COMMON_CHANCE)) {
-            AbstractCardModifier modifier;
-            switch (AbstractDungeon.cardRng.random(4)) {
-                case 0:
-                    modifier = new SharpModifier(true);
-                    break;
-                case 1:
-                    modifier = new ToughModifier(true);
-                    break;
-                case 2:
-                    modifier = new ExposingModifier(true);
-                    break;
-                case 3:
-                    modifier = new CripplingModifier(true);
-                    break;
-                default:
-                    modifier = null;
-                    break;
-            }
-            if (modifier != null) {
-                CardModifierManager.addModifier(card, modifier);
+            List<AbstractManaSurgeModifier> commonModifierList = new ArrayList<>(getPositiveCommonModifierList(true));
+            AbstractManaSurgeModifier selectedModifier = commonModifierList.get(AbstractDungeon.cardRng.random(commonModifierList.size()));
```

Comment:
We can optimize this a bit
`CardModifierManager.addModifier(card, Wiz.getRandomItem(commonModiiferList, AbstractDungeon.cardRng));`

Reply:
i think i've done it but im getting Argument `'Wiz.getRandomItem(commonModifierList, AbstractDungeon.cardRandomRng)' might be null`, is that fine??

(changed it to cardRandomRng for the temporary bc it's in combat too is that correct?)